Update developing-locally.md (#11299)
PostHog/posthog.com: contents/handbook/engineering/developing-locally.md

4. In the codespace, open a terminal window and run `docker compose -f docker-compose.dev.yml up`.
5. In another terminal, run `pnpm i` (and use the same terminal for the following commands)
6. Then run `uv sync`

>    1. If the terminal returns `uv: command not found` error then that means it is not installed, run `python -m pip install uv` then try `uv sync` again. 
> 8. Now run `DEBUG=1 ./bin/migrate` and then `./bin/start`.
>    1. Incase you get an error `sqlx: command not found`:
>    1. Install cargo `curl https://sh.rustup.rs -sSf | sh`
>    1. Install sqlx `argo install sqlx-cli` then try the command again

Reviewer: ```suggestion
   1. If you get the `uv: command not found` error, install `uv` with `python -m pip install uv` then try `uv sync` again. 
8. Now run `DEBUG=1 ./bin/migrate` and then `./bin/start`.
   1. If you get the `sqlx: command not found` error, install `cargo` with `curl https://sh.rustup.rs -sSf | sh` and `sqlx` with `argo install sqlx-cli` then try the command again
```
